tests/rptest/tests/node_restart_probe_test.py

```diff
@@ -79,10 +80,12 @@ def toggle(inject: bool):
             toggle(False)
 
     def produce_to_all_partitions(self, acks):
-        for topic in self.topics:
+        for topic_no, topic in enumerate(self.topics):
             self.redpanda.logger.debug(f"producing to {topic.name}")
-            produce_bytes = (self.PRODUCE_BYTES +
-                             self.PRODUCE_BYTES_JITTER * random.uniform(-1, 1))
+            # produce shorter data first to increase discrepancy
+            # between partitions
```

Comment:
I think the comment should explain why we need this. Also, partitions -> topics.